src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -123,12 +123,8 @@ export default class FiatConnectQuote extends NormalizedQuote {
     )
   }
 
-  // TODO: Navigate to the Review screen if a fiatAccount is already saved
-  navigate(flow: CICOFlow): void {
-    navigate(Screens.FiatDetailsScreen, {
-      quote: this,
-      flow,
-    })
+  navigate(dispatch: Dispatch): void {
+    dispatch(selectFiatConnectQuote({ quote: this }))
```

Comment:
Do we need to show a spinner while the GET accounts call is being made to the provider? What if the user performs some navigation of their own while the request is being sent but before the saga calls `navigate`?

Reply:
also good call 👍